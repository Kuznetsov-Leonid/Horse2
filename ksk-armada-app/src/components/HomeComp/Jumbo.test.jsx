import { render, screen } from '@testing-library/react';
import { 
    JumboPrise,  
    JumboEducationHorseRide, 
    JumboHorseRide, 
    JumboHoldUpWithHorse, 
    JumboPhotoWithHorse 
    } from './Jumbotrone';

describe('Jumbotrone components', () => {
    //Простой рендер компонентов
    it('JumboPrise render', () => {
        render(<JumboPrise />);
        expect(screen.queryByRole('JumboPrise')).toBeNull();
    });
    
    it('JumboEducationHorseRide render', () => {
        render(<JumboEducationHorseRide />);
        expect(screen.queryByRole('JumboEducationHorseRide')).toBeNull();
    });

    it('JumboHorseRide render', () => {
        render(<JumboHorseRide />);
        expect(screen.queryByRole('JumboHorseRide')).toBeNull();
    });

    it('JumboHoldUpWithHorse render', () => {
        render(<JumboHoldUpWithHorse />);
        expect(screen.queryByRole('JumboHoldUpWithHorse')).toBeNull();
    });

    it('JumboPhotoWithHorse render', () => {
        render(<JumboPhotoWithHorse />);
        expect(screen.queryByRole('JumboPhotoWithHorse')).toBeNull();
    });
});

describe('Jumbotrone snapshot', () => {
    it('JumboPrise snapshot', () => {
        const JumboPriseSnap = render(<JumboPrise />);
        expect(JumboPriseSnap).toMatchSnapshot();
    });

    it('JumboEducationHorseRide snapshot', () => {
        const JumboEducationHorseRideSnap = render(<JumboEducationHorseRide />);
        expect(JumboEducationHorseRideSnap).toMatchSnapshot();
    });

    it('JumboHorseRide snapshot', () => {
        const JumboHorseRideSnap = render(<JumboHorseRide />);
        expect(JumboHorseRideSnap).toMatchSnapshot();
    });

    it('JumboHoldUpWithHorse snapshot', () => {
        const JumboHoldUpWithHorseSnap = render(<JumboHoldUpWithHorse />);
        expect(JumboHoldUpWithHorseSnap).toMatchSnapshot();
    });

    it('JumboPhotoWithHorse snapshot', () => {
        const JumboPhotoWithHorseSnap = render(<JumboPhotoWithHorse />);
        expect(JumboPhotoWithHorseSnap).toMatchSnapshot();
    });
});


