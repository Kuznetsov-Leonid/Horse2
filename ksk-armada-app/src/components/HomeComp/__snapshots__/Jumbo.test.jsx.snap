// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jumbotrone snapshot JumboEducationHorseRide snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bczRLJ bIDtHO"
      >
        <div
          class="JumboPrise condComp"
        >
          <div
            class="container"
          >
            <div
              class="overlay card"
            >
              <div
                class="row"
              >
                <h1>
                  Обучение верховой езде
                </h1>
                <div
                  class="col"
                >
                  <h6>
                    Обучаем с основ до профессионального уровня, даже если Вы до этого никогда не были в седле. Уроки проходят в группах или индивидуально, а лошадки подбираются в соответствии с возрастом, весом и навыками всадника.
                  </h6>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bczRLJ bIDtHO"
    >
      <div
        class="JumboPrise condComp"
      >
        <div
          class="container"
        >
          <div
            class="overlay card"
          >
            <div
              class="row"
            >
              <h1>
                Обучение верховой езде
              </h1>
              <div
                class="col"
              >
                <h6>
                  Обучаем с основ до профессионального уровня, даже если Вы до этого никогда не были в седле. Уроки проходят в группах или индивидуально, а лошадки подбираются в соответствии с возрастом, весом и навыками всадника.
                </h6>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Jumbotrone snapshot JumboHoldUpWithHorse snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bczRLJ bIDtHO"
      >
        <div
          class="JumboPrise condComp"
        >
          <div
            class="container"
          >
            <div
              class="overlay card"
            >
              <div
                class="row"
              >
                <h1>
                  Постой лошадей
                </h1>
                <div
                  class="col"
                >
                  <h6>
                    Создавать комфортные условия для жизни лошадей — наша работа!
                  </h6>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bczRLJ bIDtHO"
    >
      <div
        class="JumboPrise condComp"
      >
        <div
          class="container"
        >
          <div
            class="overlay card"
          >
            <div
              class="row"
            >
              <h1>
                Постой лошадей
              </h1>
              <div
                class="col"
              >
                <h6>
                  Создавать комфортные условия для жизни лошадей — наша работа!
                </h6>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Jumbotrone snapshot JumboHorseRide snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bczRLJ bIDtHO"
      >
        <div
          class="JumboPrise condComp"
        >
          <div
            class="container"
          >
            <div
              class="overlay card"
            >
              <div
                class="row"
              >
                <h1>
                  Конные прогулки
                </h1>
                <div
                  class="col"
                >
                  <h6>
                    Наслаждайтесь природой и общением с животными. Лучший выбор для семейного отдыха и встреч с друзьями.
                  </h6>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bczRLJ bIDtHO"
    >
      <div
        class="JumboPrise condComp"
      >
        <div
          class="container"
        >
          <div
            class="overlay card"
          >
            <div
              class="row"
            >
              <h1>
                Конные прогулки
              </h1>
              <div
                class="col"
              >
                <h6>
                  Наслаждайтесь природой и общением с животными. Лучший выбор для семейного отдыха и встреч с друзьями.
                </h6>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Jumbotrone snapshot JumboPhotoWithHorse snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bczRLJ bIDtHO"
      >
        <div
          class="JumboPrise condComp"
        >
          <div
            class="container"
          >
            <div
              class="overlay card"
            >
              <div
                class="row"
              >
                <h1>
                  Фотосессии с лошадьми
                </h1>
                <div
                  class="col"
                >
                  <h6>
                    Чтoбы вaши впeчaтлeния сoхрaнились нe тoлькo в вaшeй пaмяти, мы прeдлaгaeм прoфeссиoнaльную фoтoсeссию с лoшaдьми.
                  </h6>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bczRLJ bIDtHO"
    >
      <div
        class="JumboPrise condComp"
      >
        <div
          class="container"
        >
          <div
            class="overlay card"
          >
            <div
              class="row"
            >
              <h1>
                Фотосессии с лошадьми
              </h1>
              <div
                class="col"
              >
                <h6>
                  Чтoбы вaши впeчaтлeния сoхрaнились нe тoлькo в вaшeй пaмяти, мы прeдлaгaeм прoфeссиoнaльную фoтoсeссию с лoшaдьми.
                </h6>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Jumbotrone snapshot JumboPrise snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bczRLJ bIDtHO"
      >
        <div
          class="JumboPrise condComp"
        >
          <div
            class="container"
          >
            <div
              class="overlay card"
            >
              <div
                class="row"
              >
                <h1>
                  Спортивные мероприятия
                </h1>
                <div
                  class="col"
                >
                  <h6>
                    На нашей базе регулярно проходят соревнования по конкуру и конной выездке.
                  </h6>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bczRLJ bIDtHO"
    >
      <div
        class="JumboPrise condComp"
      >
        <div
          class="container"
        >
          <div
            class="overlay card"
          >
            <div
              class="row"
            >
              <h1>
                Спортивные мероприятия
              </h1>
              <div
                class="col"
              >
                <h6>
                  На нашей базе регулярно проходят соревнования по конкуру и конной выездке.
                </h6>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
